nodetypes/radon.nodes.monitoring/PushGateway/NodeType.tosca

```diff
@@ -0,0 +1,96 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
+
+node_types:
+  radon.nodes.monitoring.PushGateway:
+    derived_from: radon.nodes.docker.DockerApplication
+    metadata:
+      targetNamespace: "radon.nodes.monitoring.PushGateway"
+      abstract: "false"
+      final: "false"
+    properties:
+      user_email:
+        type: string
+        default: { get_property: [SELF, user_email] }
+      consul_ip:
+        type: string
+        default: { get_property: [SELF, consul_ip] }
+      pushgateway_service_port:
+        type: string
+        default: { get_property: [SELF, pushgateway_service_port] }
```

Comment:
Same as above, can we use `get_input` here?

Reply:
Done!